shark_turbine/kernel/functional/README.md

```diff
@@ -0,0 +1,99 @@
+# TKF: A Wave Kernel Language for GPUs
+
+TKF is a high-level programming language designed to simplify the development of GPU micro-kernels by abstracting over intricate details of GPU hardware. It allows developers to write efficient micro-kernels, focusing on core computations while inferring the required data transfers and indexing automatically. By leveraging a wave-based programming model, TKF enables expressing programs leveraging coalesced memory accesses effortlessly.
```

Comment:
Could you modify this to include the mma intrinsics as well , like - TKF enables expressing programs leveraging coalesced memory accesses and matrix multiplication intrinsics effortlessly.